team_red/models/model.py

```diff
@@ -21,7 +21,8 @@ def model_post_init(self, __context: Any) -> None:  # noqa: ANN401
     @computed_field  # type: ignore[misc]
     @property
     def parameters(self) -> List[str]:
-        return [fn for _, fn, _, _ in Formatter().parse(self.text) if fn is not None]
+        unqiue = {fn for _, fn, _, _ in Formatter().parse(self.text) if fn is not None}
```

Comment:
'unqiue' is probably not the correct spelling. maybe use something more expressive like `field_names`.